scripts/sanity_check.sh

```diff
@@ -642,37 +647,65 @@ check_network() {
     fi
 
     echo "---------------------- Network ------------------"
-    curl localhost:8837/v1/config/interfaces -o /tmp/scc_interfaces &>/dev/null
+    curl localhost:8837/v1/config/interfaces -s -o /tmp/scc_interfaces
     INTFS=$(cat /tmp/scc_interfaces | jq 'keys' | jq -r .[])
-
-    echo "Interface,Name,UUID,IPv4,Gateway,IPv6,Gateway6,DNS,vpnClient" >/tmp/scc_csv
+    curl localhost:8837/v1/config/active -s -o /tmp/scc_config
+    DNS=$(cat /tmp/scc_config | jq '.dns')
+    # read LAN DNS as '|' seperated string into associative array DNS_CONFIG
+    declare -A DNS_CONFIG
+    cat /tmp/scc_config |
+      jq '.dns | to_entries | map(select(.value.nameservers))[] | .key, (.value.nameservers | join("|"))' |
+      while mapfile -t -n 2 ARY && ((${#ARY[@]})); do
+        DNS_CONFIG[${ARY[0]}]=${ARY[1]}
+      done
+
+    >/tmp/scc_csv
     for INTF in $INTFS; do
       jq -rj ".[\"$INTF\"] | if (.state.ip6 | length) == 0 then .state.ip6 |= [] else . end | [\"$INTF\", .config.meta.name, .config.meta.uuid[0:8], .state.ip4, .state.gateway, (.state.ip6 | join(\"|\")), .state.gateway6, (.state.dns // [] | join(\";\"))] | @csv" /tmp/scc_interfaces >>/tmp/scc_csv
       echo ',"'$(redis-cli hget policy:network:$(jq -rj ".[\"$INTF\"].config.meta.uuid" /tmp/scc_interfaces) vpnClient |  jq -r 'select(.state == true) | .profileId')'"' >> /tmp/scc_csv
     done
 
-    > /tmp/scc_csv_multline
+    echo "Interface,Name,UUID,IPv4,Gateway,IPv6,Gateway6,DNS,vpnClient" >/tmp/scc_csv_multline
     while read -r LINE; do
       mapfile -td ',' COL <<< $LINE
+      # read multi line fields into array
       mapfile -td '|' IP6 < <(echo ${COL[5]}| xargs) #remove quotes with xargs
-      if [[ ${#IP6[@]} -gt 1 ]]; then
-        for IDX in "${!IP6[@]}"; do
-          if [ $IDX -eq 0 ]; then
-            echo -n "${COL[0]},${COL[1]},${COL[2]},${COL[3]},${COL[4]},\"${IP6[0]}\",${COL[6]},${COL[7]},${COL[8]}" >> /tmp/scc_csv_multline
-          else
-            echo '"","","","","","'${IP6[$IDX]}'","","",""' >> /tmp/scc_csv_multline
-          fi
-        done
-      else
-        echo $LINE >> /tmp/scc_csv_multline
-      fi
+      mapfile -td '|' DNS < <(echo ${DNS_CONFIG["${COL[0]}"]}| xargs)
+      # echo ${COL[0]}
+      # echo "ip${#IP6[@]} dns${#DNS[@]}"
+      # echo ${DNS_CONFIG["${COL[0]}"]}
+      # echo ${IP6[@]}
+      # echo ${DNS[@]}
+
+      LINE_COUNT=$(( "${#IP6[@]}" > "${#DNS[@]}" ? "${#IP6[@]}" : "${#DNS[@]}" ));
+      for (( IDX=0; IDX < $LINE_COUNT; IDX++ )); do
+        # echo $IDX
+        if [[ ${#IP6[@]} > $IDX ]]; then
```

Comment:
For numeric comparison, use `-gt` instead of `>` ?
```
pi@firewalla:~ (G) $ if [[ 3 > 12 ]]; then echo h; else echo l; fi
h
pi@firewalla:~ (G) $ if [[ 3 -gt 12 ]]; then echo h; else echo l; fi
l
```